chapter4/section1.md

```diff
@@ -9,6 +9,8 @@ Before code is merged it needs to comply with a number of guidelines:
 2. Right formatting; code should follow the Coding Conventions (see 3.1.2)
 3. Fixed/added unit tests where applicable
 4. Javadocs added where applicable
+5. Accepting pull-requests with SonarQube reports "Critical" and "Major" are not allowed
```

Comment:
My understanding was that "Blocker" and "Critical" SonarQube issues are not allowed (except for obvious false positives).